tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_component.scss

```diff
@@ -78,3 +78,9 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
     color: mat.get-color-from-palette($tb-dark-foreground, secondary-text);
   }
 }
+
+// The mat-tab-body-wrapper should grow to the full height to surround it's
```

Comment:
it's => its. "its" is the possessive form of "it". "it's" is a contraction for "it is".

Reply:
I am pretty sure I have used this incorrectly my entire life. Fixed.